extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java

```diff
@@ -223,79 +217,78 @@ public void setSecretProvider(Provider secretProvider) {
         }
 
         /**
-         * Supports the client authentication 'client_secret_jwt' and `private_key_jwt` methods which involve sending a JWT
-         * token
-         * assertion signed with either a client secret or private key.
+         * Supports the client authentication `client_secret_jwt` and `private_key_jwt` methods, which involves sending a JWT
+         * token assertion signed with a client secret or private key.
          *
          * @see <a href=
          *      "https://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication">https://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication</a>
          */
         @ConfigGroup
         public static class Jwt {
             /**
-             * If provided, indicates that JWT is signed using a secret key
+             * Optional. If provided, indicates that JWT is signed using a secret key.
              */
             @ConfigItem
             public Optional<String> secret = Optional.empty();
 
             /**
-             * If provided, indicates that JWT is signed using a secret key provided by Secret CredentialsProvider
+             * If provided, indicates that JWT is signed using a secret key provided by Secret CredentialsProvider.
              */
             @ConfigItem
             public Provider secretProvider = new Provider();
 
             /**
-             * If provided, indicates that JWT is signed using a private key in PEM or JWK format. You can use the
+             * Optional. If provided, indicates that JWT is signed using a private key in PEM or JWK format. You can use the
```

Comment:
I have to admit, I'm not sure highlighting `Optional` everywhere works great.

Reply:
Thanks. I've replaced 3 instances of `Optional. If provided,` with `If provided,`. Please let me know if there are other cases where the "Optional" label doesn't help.

Reply:
Thanks again, Sergey. After our conversation, I removed all the instances of `Optional.` that I had added.